packages/common/src/types/filter.test.ts

```diff
@@ -19,7 +20,7 @@ describe('compress and uncompress dashboard filters', () => {
                 fieldId: 'payments_payment_method',
                 tableName: 'payments',
             },
-            tileTargets: {},
+            tileTargets: {} as Record<string, DashboardTileTarget>,
```

Comment:
is this change in the test necessary for tsoa ? This type hasn't changed, right ?